pyprland/plugins/pyprland.py

```diff
@@ -9,6 +9,13 @@ class Extension(Plugin):
     async def init(self):
         "initializes the plugin"
         state.active_window = ""
+        version = (await self.hyprctlJSON("version"))["tag"]
+        try:
+            state.hyprland_version = VersionInfo(
+                (int(i) for i in version[1:].split("."))
+            )
+        except Exception:
+            pass
```

Comment:
In case of error can you log it instead of passing ?

Reply:
I will update this